product_docs/docs/pgd/5/choosing_server.mdx

```diff
@@ -27,10 +27,10 @@ The following table lists features of EDB Postgres Distributed that are dependen
 | [Group Commit](durability/group-commit/)                                    | N          | Y                     | 14+                   |
 | [Commit At Most Once (CAMO)](durability/camo/)                      | N          | Y                     | 14+                   |
 | [Eager Conflict Resolution](consistency/eager/)                       | N          | Y                     | 14+                   |
-| [Lag Control](durability/lag-control/)                                     | N          | Y                     | 14+                   |
+| [Lag control](durability/lag-control/)                                     | N          | Y                     | 14+                   |
```

Comment:
```suggestion
| [Lag Control](durability/lag-control/)                                     | N          | Y                     | 14+                   |
```

Lag Control is a category and feature of Commit Scopes, so I think should be upper case. (Unlike Lag tracking which is the tracking of lag and not a feature per se).